Rules.js

```diff
@@ -1,54 +0,0 @@
-
```

Comment:
Obsoleted complete Rules.js file
- migrated functionality to new Superclass of Cell & Card following the responsability
- eliminated case-switches using delegation